lib/config/config-ops.js

```diff
@@ -30,28 +30,28 @@ var RULE_SEVERITY_STRINGS = ["off", "warn", "error"],
 // Public Interface
 //------------------------------------------------------------------------------
 
-module.exports = {
+module.exports = class ConfigOps {
```

Comment:
May want to consider not making this into a class. A class whose only members are static functions is just a fancy way of saying `object`.

Personally I find all the extra `static` keywords to be noisy.

Thoughts?